docs/source/containerization/kubernetes.mdx

```diff
@@ -1,246 +1,274 @@
-our helm chart and store it in the same OCI registry in which we store our
-router docker images.
 
-You can use helm to install charts from an OCI registry as follows:
+## Get the router Helm chart
+
+The Helm chart for each release of the Apollo Router is available from the router's GitHub repository in `<repo>/helm/chart/router`. 
+
+> For example, the Helm chart of the latest release of the router is available [here](https://github.com/apollographql/router/tree/v1.31.1/helm/chart/router).
+
+Since the router v0.14.0, Apollo has also released a Helm chart for the router as an [Open Container Initiative (OCI)](https://helm.sh/docs/topics/registries/) image in the GitHub Container registry. 
+
+> The path to the OCI router chart is `oci://ghcr.io/apollographql/helm-charts/router`.
+
+<ExpansionPanel title="Click to expand example chart for router v1.31.0">
+
+The values of the Helm chart for Apollo Router v1.31.0 in the GitHub container repository, as output by the `helm show` command:
 
 ```bash
-helm install --set router.configuration.telemetry.metrics.prometheus.enabled=true --set managedFederation.apiKey="REDACTED" --set managedFederation.graphRef="REDACTED" --create-namespace --namespace router-deploy router-test oci://ghcr.io/apollographql/helm-charts/router --version 1.0.0-rc.8 --values router/values.yaml
+helm show values oci://ghcr.io/apollographql/helm-charts/router
 ```
 
-For more details about using helm with OCI based registries, see [here](https://helm.sh/docs/topics/registries/)
+<HelmShowOutput/>
 
-#### Using helm chart from your filesystem
+</ExpansionPanel>
 
-You would run this command from "repo"/helm/chart directory.
+## Separate configurations per environment
 
-(where "repo" is the directory containing your checked out router github repository)
+To support your different deployment configurations for different environments (development, staging, production, etc.), Apollo recommends separating your configuration values into separate files (`values.yaml`):
 
-```bash
-helm install --set router.configuration.telemetry.metrics.prometheus.enabled=true --set managedFederation.apiKey="REDACTED" --set managedFederation.graphRef="REDACTED" --create-namespace --namespace router-deploy router-test router --values router/values.yaml
+- A **common** file, which contains values that apply across all environments.
+- A unique **environment** file per environment, which includes and overrides the values from the common chart while adding new environment-specific values.
+
+### Common value configuration
+
+Configuring a `values.yaml` file for settings common across environments.
+
+<ExpansionPanel title="Example router Helm chart for common configuration across environments">
+
+<RouterCommonConfig/>
+
+</ExpansionPanel>
+
+(TODO: explain `fullnameOverride`)
+
+Setting router command-line arguments and YAML config file keys/values. 
+
+```yaml title="values.yaml"
+router:
+  router:
```

Comment:
Everywhere where you have:
```
router:
  router:
```
The first `router:` needs to be deleted and all the rest of the text needs to be left shifted by 2 bytes.

REASON: We are using an in-house chart which "wraps" the main router chart. We do that to make it simple to deploy `Rhai` and configure `istio`. I don't think we are committing to documenting that here, since it's Apollo specific, but if we are doing that, then we need to decide how to document it.